R/Features-missing-data.R

```diff
@@ -85,16 +124,17 @@ setMethod("zeroIsNA", c("SummarizedExperiment", "missing"),
 ##' @rdname Features-missing-data
 setMethod("zeroIsNA", c("Features", "missing"),
           function(object, i) {
-              for (i in seq_len(length(object)))
-                  object[[i]] <- zeroIsNA(object[[i]])
-              object
+            for (i in seq_len(length(object)))
+              object[[i]] <- zeroIsNA(object[[i]])
+            object
           })
 
 ##' @rdname Features-missing-data
 setMethod("zeroIsNA", c("Features", "integer"),
```

Comment:
Do we need integer, or is numeric not enough?

Reply:
I tested removing `setMethod("zeroIsNA", c("Features", "integer"){...})`, but this causes an infinite call between the generic method (from `AllGenerics.R`) and the method implementation (from `Features-mising-data.R`). See the traceback:
```
[...]
8: zeroIsNA(object, as.integer(i)) at AllGenerics.R#12
7: zeroIsNA(object, as.integer(i)) at Features-missing-data.R#133
6: zeroIsNA(object, as.integer(i)) at AllGenerics.R#12
5: zeroIsNA(object, as.integer(i)) at Features-missing-data.R#133
4: zeroIsNA(object, as.integer(i)) at AllGenerics.R#12
3: zeroIsNA(object, as.integer(i)) at Features-missing-data.R#133
2: zeroIsNA(ft0, 1) at AllGenerics.R#12
1: zeroIsNA(ft0, 1)
```
I think it is safer to leave it, unless you see an obvious fix.